web/src/utils/units.ts

```diff
@@ -0,0 +1,22 @@
+export enum PowerUnits {
+  WATTS = 'W',
+  KILOWATTS = 'kW',
+  MEGAWATTS = 'MW',
+  GIGAWATTS = 'GW',
+  TERAWATTS = 'TW',
+  PETAWATTS = 'PW',
+}
+
+export enum EnergyUnits {
+  WATT_HOURS = 'Wh',
+  KILOWATT_HOURS = 'kWh',
+  MEGAWATT_HOURS = 'MWh',
+  GIGAWATT_HOURS = 'GWh',
+  TERAWATT_HOURS = 'TWh',
+  PETAWATT_HOURS = 'PWh',
+}
+
+export enum CarbonUnits {
+  GRAMS_CO2EQ_PER_WATT_HOUR = 'gCO₂eq/kWh',
+  CO2EQ_PER_HOUR = 'CO₂eq/h',
```

Comment:
I thought about that as well but then it would also mean we would have to scale up the other value to megatons and gigatons to avoid a excessive amount of important figures. I think we should evaluate those units in particular when we are ready to test them / have if fully shaped.